docs/swarmmode.md

```diff
@@ -0,0 +1,73 @@
+# Microsoft Azure Container Service Engine - Swarm Mode Walkthrough
+
+## Deployment
+
+Here are the steps to deploy a simple Swarm cluster:
+
+1. [install acs-engine](acsengine.md#downloading-and-building-acs-engine)
+2. [generate your ssh key](ssh.md#ssh-key-generation)
+3. edit the [Swarm Mode example](../examples/swarmmode.json) and fill in the blank strings
+4. [generate the template](acsengine.md#generating-a-template)
+5. [deploy the output azuredeploy.json and azuredeploy.parameters.json](../README.md#deployment-usage)
+
+## Walkthrough
+
+Once your Swarm Mode cluster has been deployed you will have a resource group containing:
+
+1. a set of 1,3, or 5 masters in a master availability set.  Each master's SSH can be accessed via the public dns address at ports 2200..2204

```

Comment:
Done. Tested with Swarm Mode classicMode and non classicMode. SSHing on port 22 and 2200 works for both the templates with the update.